### DISCUSSION TITLE
[1190970]房产大亨2 本体 密钥+清单分享

### DISCUSSION THREAD

**Discussion by alex110119** [pjy612/SteamManifestCache#82] [Share]
AppID:1190970
"Depots"
{
   "1190971"
       {
              						"DecryptionKey" "a6202777d416dcf4d2f36cc5f7a69f90cca6e1bfbde6e27eaa4e97e86c7d3196"
      }

}
联系方式：alex12110@qq.com
内测code：AC4FB80E9A988D1D
[HouseFlipper.zip](https://github.com/pjy612/SteamManifestCache/files/13699398/HouseFlipper.zip)

**Comment by pjy612** [comment]
查收
